hosting/single/Dockerfile

```diff
@@ -1,22 +1,22 @@
-FROM node:14-slim as build
+FROM node:16-slim as build
 
 # install node-gyp dependencies
 RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends apt-utils cron g++ make python
 
 # add pin script
 WORKDIR /
-ADD scripts/pinVersions.js scripts/cleanup.sh ./
+ADD scripts/cleanup.sh ./
 RUN chmod +x /cleanup.sh
 
 # build server
```

Comment:
We will be already using the built version, right? If so, this comment needs to be removed and we can change this to be `yarn install --frozen-lockfile --production=true`

Reply:
Actually, we need to do the same to the worker/server dockerfile

Reply:
Fixed - added for server/worker as well.